tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -32,7 +36,7 @@ prepare: |
          b: B
        system:
          service:
-           rsyslog:
+           $SERVICE:
```

Comment:
we want to test also:

${SERVICE}.disable: true

We probably want to have a 2nd test about only ssh, and then use  variants for the two layouts for the default settings there.

Reply:
sure, I'll make that change. Thanks for reviewing